LICENSE

```diff
@@ -175,24 +175,13 @@
 
    END OF TERMS AND CONDITIONS
```

Comment:
It probably doesnt matter, but do we need lines 1-176, I think most repos ive seen just have lines 178-190

Reply:
@dr3 I'd say so as I'm following http://www.apache.org/licenses/LICENSE-2.0 for exactly how it looks. Is there an example to strongly suggest otherwise?

Reply:
@radiocontrolled It probably doesnt matter, but that link says `To apply the Apache License to your work, attach the following boilerplate notice`, implying you only need the boilerplate last bit, but idm, your call

Reply:
@dr3 ok, it's updated